app/ide-desktop/lib/content/src/index.ts

```diff
@@ -964,13 +975,15 @@ async function runEntryPoint(config: Config) {
 }
 
 API.main = async function (inputConfig: any) {
-    const urlParams = new URLSearchParams(window.location.search)
-    // @ts-ignore
-    const urlConfig = Object.fromEntries(urlParams.entries())
-
-    const config = Config.default()
-    config.updateFromObject(inputConfig)
-    config.updateFromObject(urlConfig)
+        const config = profiling.task.measure('load_config', () => {
+        const urlParams = new URLSearchParams(window.location.search)
+        // @ts-ignore
+        const urlConfig = Object.fromEntries(urlParams.entries())
+        const config = Config.default()
+        config.updateFromObject(inputConfig)
+        config.updateFromObject(urlConfig)
+        return config
+    })
```

Comment:
I believe the indentation is broken here – I'm wondering why - is this code listed / autoformatted?